common/src/main/java/com/wynntils/models/abilities/ShamanTotemModel.java

```diff
@@ -116,6 +148,11 @@ public void onTimerRename(TextDisplayChangedEvent.Text event) {
         int parsedTime = Integer.parseInt(m.group("time"));
         int timerId = textDisplay.getId();
 
+        if (orphanedTimers.getOrDefault(timerId, 0) >= 3) {
+            // check every other second to find the totem to avoid performance problems
+            if (parsedTime % 2 == 0) return;
+        }
+
```

Comment:
Given my comment above, do we need this? Since only error/warn logging took time, not the actual matching code, this is incorrect optimization, as rather, the logging should be reduced.